2022-10-18T16:43:19 (svs.sh:205:main) ./svs.sh --lang jp --stage 6 --stop_stage 9 --local_data_opts --stage 0 --feats_type raw --pitch_extract None --fs 24000 --fmax 7600 --fmin 80 --n_fft 2048 --n_shift 300 --win_length 1200 --token_type phn --g2p pyopenjtalk --cleaner none --train_config conf/tuning/train_xiaoice.yaml --inference_config conf/decode.yaml --train_set tr_no_dev --valid_set dev --test_sets dev eval --score_feats_extract syllable_score_feats --srctexts data/tr_no_dev/text --svs_exp exp/xiaoice_gt_2 --audio_format wav
2022-10-18T16:43:19 (svs.sh:647:main) Stage 6: SVS Training: train_set=dump/raw/tr_no_dev, valid_set=dump/raw/dev
2022-10-18T16:43:19 (svs.sh:843:main) Generate 'exp/xiaoice_gt_2/run.sh'. You can resume the process from stage 6 using this script
2022-10-18T16:43:19 (svs.sh:848:main) SVS training started... log: 'exp/xiaoice_gt_2/train.log'
2022-10-18 16:43:20,354 (launch:94) INFO: /root/anaconda3/envs/espnet/bin/python3 /data1/wyn/espnet/espnet2/bin/launch.py --cmd 'run.pl --name exp/xiaoice_gt_2/train.log' --log exp/xiaoice_gt_2/train.log --ngpu 1 --num_nodes 1 --init_file_prefix exp/xiaoice_gt_2/.dist_init_ --multiprocessing_distributed true -- python3 -m espnet2.bin.svs_train --use_preprocessor true --token_type phn --token_list data/token_list/phn_pyopenjtalk_jp/tokens.txt --syb_token_list data/token_list/phn_pyopenjtalk_jp/tokens.txt_syb --non_linguistic_symbols none --cleaner none --g2p pyopenjtalk --normalize global_mvn --resume true --init_param --ignore_init_mismatch false --fold_length 150 --fold_length 240000 --output_dir exp/xiaoice_gt_2 --config conf/tuning/train_xiaoice.yaml --score_feats_extract syllable_score_feats --score_feats_extract_conf fs=24000 --score_feats_extract_conf n_fft=2048 --score_feats_extract_conf win_length=1200 --score_feats_extract_conf hop_length=300 --feats_extract fbank --feats_extract_conf n_fft=2048 --feats_extract_conf hop_length=300 --feats_extract_conf win_length=1200 --pitch_extract None --feats_extract_conf fs=24000 --feats_extract_conf fmin=80 --feats_extract_conf fmax=7600 --feats_extract_conf n_mels=80 --train_data_path_and_name_and_type dump/raw/tr_no_dev/text,text,text --train_data_path_and_name_and_type dump/raw/tr_no_dev/wav.scp,singing,sound --train_data_path_and_name_and_type dump/raw/tr_no_dev/label,label,duration --train_data_path_and_name_and_type dump/raw/tr_no_dev/xml.scp,midi,midi --train_shape_file exp/svs_stats_raw_phn_pyopenjtalk_jp/train/text_shape.phn --train_shape_file exp/svs_stats_raw_phn_pyopenjtalk_jp/train/singing_shape --train_shape_file exp/svs_stats_raw_phn_pyopenjtalk_jp/train/durations_shape --train_shape_file exp/svs_stats_raw_phn_pyopenjtalk_jp/train/score_shape --valid_data_path_and_name_and_type dump/raw/dev/text,text,text --valid_data_path_and_name_and_type dump/raw/dev/wav.scp,singing,sound --valid_data_path_and_name_and_type dump/raw/dev/label,label,duration --valid_data_path_and_name_and_type dump/raw/dev/xml.scp,midi,midi --valid_shape_file exp/svs_stats_raw_phn_pyopenjtalk_jp/valid/text_shape.phn --valid_shape_file exp/svs_stats_raw_phn_pyopenjtalk_jp/valid/singing_shape --valid_shape_file exp/svs_stats_raw_phn_pyopenjtalk_jp/valid/durations_shape --valid_shape_file exp/svs_stats_raw_phn_pyopenjtalk_jp/valid/score_shape --normalize_conf stats_file=exp/svs_stats_raw_phn_pyopenjtalk_jp/train/feats_stats.npz
2022-10-18 16:43:20,403 (launch:348) INFO: log file: exp/xiaoice_gt_2/train.log
2022-10-19T19:56:28 (svs.sh:888:main) Stage 7: Decoding: training_dir=exp/xiaoice_gt_2
2022-10-19T19:56:29 (svs.sh:928:main) Generate 'exp/xiaoice_gt_2/decode_latest/run.sh'. You can resume the process from stage 7 using this script
2022-10-19T19:56:29 (svs.sh:983:main) Decoding started... log: 'exp/xiaoice_gt_2/decode_latest/dev/log/svs_inference.*.log'
mv: cannot stat 'exp/xiaoice_gt_2/decode_latest/dev/log/output.1/wav/*.wav': No such file or directory
